src/components/RequestPanel/RequestPanel.js

```diff
@@ -196,7 +196,7 @@ function RequestPanel() {
                 await ensureRequestConfirmed(requestProcess)
 
                 // Execute callback function if required
-                if (request.callback) {
+                if (typeof request?.callback === 'function') {
```

Comment:
Oh this is interesting, what does this do if `request` is null? Would it do a `typeof null`?

Reply:
Ah actually i think we don't need the `?` since we are iterating on the requests. a `typeof request.callback === 'function'` should do. If there's no callback it will be `typeof undefined === 'function'`